.travis.yml

```diff
@@ -9,8 +9,3 @@ script:
 
 # after_success:
 #   - bash <(curl -s https://codecov.io/bash) || echo "Failed to report code coverage!"
-
-cache:
```

Comment:
Why to delete this, is this related to pouchlinter?

Reply:
No. I think this config is for previous Java version. Do we still need that?

Reply:
No, we don't.